_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -102,6 +154,31 @@ Mathematical symbols cannot be substituted for text as [explicitly mentioned in
 <a href="/" aria-label="Proof of two multiplied by two is four">Proof of 2&times;2=4</a>
 ```
 
+#### Failed Example 4
+
+The full [visible][] [label][] is split out across the [accessible name][] and is therefore not [included](#included-characters) in it.
+
+```html
+<head>
+	<style>
+		.sr-only {
+			position: absolute;
+			left: -10000px;
+			top: auto;
+			width: 1px;
+			height: 1px;
+			overflow: hidden;
+		}
+	</style>
+</head>
+
+<body>
+	<a href="https://github.com/features/actions"
+		>Link <span class="sr-only">that opens a new tab</span> to the homepage</a
+	>
+</body>
+```
+
 ### Inapplicable
```

Comment:
Inapplicable example 4 uses a definition that is not used in the applicability (or the expectation). It would be better to replace it with definitions used in the applicability.